help/destinations/catalog/crm/microsoft-dynamics-365.md

```diff
@@ -0,0 +1,213 @@
+
+Read [Activate profiles and segments to streaming segment export destinations](../../ui/activate/activate-segment-streaming-destinations.md) for instructions on activating audience segments to this destination.
+
+### Mapping considerations and example {#mapping-considerations-example}
+
+To correctly send your audience data from Adobe Experience Platform to the [!DNL Microsoft Dynamics 365] destination, you need to go through the field mapping step. Mapping consists of creating a link between your Experience Data Model (XDM) schema fields in your Platform account and their corresponding equivalents from the target destination. To correctly map your XDM fields to the [!DNL Dynamics 365] destination fields, follow these steps:
+
+1. In the Mapping step, click **[!UICONTROL Add new mapping]**, you will see a new mapping row on the screen.
+![Add new mapping](../../assets/catalog/crm/microsoft-dynamics-365/add-new-mapping.png)
+
+1. In the select source field window, when selecting the source field choose the **[!UICONTROL Select identity namespace]** category and add the mappings desired.
+![Source mapping](../../assets/catalog/crm/microsoft-dynamics-365/source-mapping.png)
+
+
+1. In the "[!UICONTROL Select target field]" window, select the target field 
+    * **[!UICONTROL Select identity namespace]**: select this option to map your source field to an identity namespace from the list.
+    ![Target mapping using contactId](../../assets/catalog/crm/microsoft-dynamics-365/target-mapping-contactid.png)
+
+    * Add the following mapping between your XDM profile schema and your [!DNL Outreach] instance:
+        |XDM Profile Schema|[!DNL Outreach] Instance| Mandatory|
+        |---|---|---|
+        |`contactId`|`contactId`| Yes |
+
+    * **[!UICONTROL Select custom attributes]**: select this option to map your source field to a custom attribute that you define in the [!UICONTROL Attribute name] field. Refer to [[!DNL Dynamics 365] documentation](https://docs.microsoft.com/en-us/dynamics365/customerengagement/on-premises/developer/entities/contact?view=op-9-1#entity-properties) for a comprehensive list of supported attributes.
+    ![Target mapping using LastName](../../assets/catalog/crm/microsoft-dynamics-365/target-mapping-lastname.png)
+
+    * For instance, depending on the values you want to update, add the following mapping between your XDM profile schema and your [!DNL Outreach] instance:
+        |XDM Profile Schema|[!DNL Dynamics 365] Instance|
+        |---|---|
+        |`person.name.firstName`|`FirstName`|
+        |`person.name.lastName`|`LastName`|
+        |`personalEmail.address`|`Email`|
+
+1. An example using these mappings is shown below:
+![Target mapping](../../assets/catalog/crm/microsoft-dynamics-365/mappings.png)
+
+> [!IMPORTANT]
```

Comment:
This note seems like it comes out of nowhere. I don't understand what this is supposed to mean. Let's add some context. What does a date or timestamp mapping look like? Why would someone do that? Let's explain this in a dedicated section.

Reply:
Line 141 :  Refer to [[!DNL Dynamics 365] documentation](https://docs.microsoft.com/en-us/dynamics365/customerengagement/on-premises/developer/entities/contact?view=op-9-1#entity-properties) for a comprehensive list of supported attributes.

These have a number of date / timestamp fields. The user can map to these fields but if the value is left empty then push to dynamics would fail.

Have moved the block just below the above reference link.

Reply:
Sorry, this still doesn't make much sense to me. The supported identity for this destination is `contactId`. Why would anyone map anything to a date or timestamp? Where did this information come from? Was this asked by PM/Engineering?

Reply:
Yes, this has been specifically pointed out by engineering. There could be multiple use cases we might want to store the date of the last communication that was sent to them etc.